casbin/util/eval.cpp

```diff
@@ -60,6 +60,25 @@ std::string ReplaceEvalWithMap(const std::string& src, std::unordered_map<std::s
     return replacedExp;
 }
 
+// GetEvalValue returns the parameters of function eval
+std::vector<std::string> GetEvalValue(const std::string& s) {
+    std::vector<std::string> rules;
```

Comment:
```suggestion
    std::vector<std::string> rules;
    rules.reserve(100);  // 100 or any arbitrary number
```

Reply:
Maybe this only needs only a few capacity for `eval()`, `10` is ok I think.